tests/wsl/install/ms_win_firstboot.pm

```diff
@@ -157,7 +157,8 @@ sub run {
     );
 
     # poweroff
-    $self->reboot_or_shutdown;
+    $self->reboot_or_shutdown(1);
```

Comment:
No for this PR, but one might consider a refactoring where we are creating `reboot` and `shutdown` functions in windowsbasetest, so that we don't have to do such cryptic calls.

Note for my future self: The 1 means that it's a reboot and not a shutdown

```
sub reboot_or_shutdown {
    my ($self, $is_reboot) = @_;
```